pkg/enum/blob_commit_map_test.go

```
@@ -0,0 +1,187 @@
+
+func TestBlobCommitMap_DeletedAndReadded(t *testing.T) {
+	skipIfNoGit(t)
+
+	tmpDir := t.TempDir()
+	initGitRepo(t, tmpDir)
+
+	content := "password = 'supersecret'"
+	writeFile(t, filepath.Join(tmpDir, "creds.txt"), content)
+	gitAddCommit(t, tmpDir, "Add creds")
+
+	os.Remove(filepath.Join(tmpDir, "creds.txt"))
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)